spec/system/school/courses/exports_spec.rb

```diff
@@ -22,10 +22,50 @@
   let(:target_group) { create :target_group, level: level }
   let!(:target) { create :target, target_group: target_group }
 
+  let!(:school_admin) { create :school_admin, school: school }
+  let!(:standing_1) { create :standing, school: school, default: true }
+  let!(:standing_2) { create :standing, school: school }
+
+  let!(:user_standing_1) do
+    create(
+      :user_standing,
+      user: student_1.user,
+      standing: standing_1,
+      creator: school_admin.user,
+      reason: "Reason 1",
+      created_at: 4.days.ago
+    )
+  end
+
+  let!(:user_standing_2) do
+    create(
+      :user_standing,
+      user: student_1.user,
+      standing: standing_2,
+      creator: school_admin.user,
+      reason: "Reason 2",
+      created_at: 3.days.ago
+    )
+  end
+
+  let!(:user_standing_3) do
+    create(
+      :user_standing,
+      user: student_1.user,
+      standing: standing_2,
+      creator: school_admin.user,
+      reason: "Reason 3",
+      created_at: 3.days.ago,
+      archived_at: 2.days.ago
+    )
+  end
+
   before do
     # Add those tags to school's list of team tags.
     school.student_tag_list.add("tag 1", "tag 2", "tag 3")
     school.save!
+
+    school.update!(configuration: { enable_standing: true })
```

Comment:
Assuming that configuration will contain only this value is unsafe. It's better to edit the configuration for this context:

```suggestion
    school.configuration = school.configuration.merge("enable_standing" => true)
    school.save!
```